test/test.yaml

```diff
@@ -4,7 +4,7 @@
 # This section holds the config fields related to the printer, such as its serial
 # connection and its frame buffer specs
 printer:
-  serial: /dev/pts/6 # Mock port opened with `python3 test/mockSerial.py`
```

Comment:
The number changes with basically every restart of mockSerial.py.
Maybe make it a CLI argument instead?